src/material/progress-bar/progress-bar.md

```diff
@@ -20,17 +20,16 @@ not necessary to indicate how long it will take should use the indeterminate ind
 In this mode the `value` property is ignored.
 
 #### Buffer
-Operations where the user wants to indicate some activity or loading from the server, 
-use the buffer indicator.
+Use the buffer indicator when want to indicate some activity or loading from the server.
```

Comment:
"Indicator to indicate" sounds weird, maybe "Use the `buffer` mode of the progress bar..."?